frontend/pnpm-lock.yaml

```diff
@@ -11,7 +11,7 @@ specifiers:
   '@types/react-file-icon': ^1.0.1
```

Comment:
Please do not run a formatter on this file. Prettier should only be ran on the `src` directory.